package.json

```diff
@@ -2,6 +2,7 @@
   "name": "MyTAPP",
   "private": true,
   "dependencies": {
+    "activeresource": "^2.0.11",
```

Comment:
This doesn't seem to be related to the Ruby activeresource library: https://www.npmjs.com/package/activeresource